src/toil/cwl/cwltoil.py

```diff
@@ -90,6 +90,10 @@
 # output object to the correct key of the input object.
 
 
+def cwltoil_is_deprecated():
+    raise RuntimeError('Please run with "toil-cwl-runner" instead of "cwltoil" (deprecated).')
```

Comment:
It's not deprecated; it's gone and doesn't work anymore. We should say it has been removed.

Reply:
@adamnovak Updated!